helmfile/values/falco.yaml.gotmpl

```diff
@@ -12,12 +12,26 @@ falco:
   grpc_output:
     enabled: true
 
+{{- if eq .Values.falco.driver.kind "module" }}
 containerSecurityContext:
-  runAsUser: 0
   privileged: true
   allowPrivilegeEscalation: true
+{{- end }}
+
+podSecurityContext:
+  runAsUser: 0
 
 driver:
+  kind: {{ .Values.falco.driver.kind }}
+  {{- if eq .Values.falco.driver.kind "ebpf" }}
+  ebpf:
+    path: {{ .Values.falco.driver.ebpf.path }}
+    hostNetwork: {{ .Values.falco.driver.ebpf.hostNetwork }}
+    leastPrivileged: true
+  {{- else if eq .Values.falco.driver.kind "modern-bpf" }}
+  modern_bpf:
+    leastPrivileged: true
+  {{- end }}
   loader:
     initContainer:
       securityContext:
```

Comment:
Is this loader used for all three kinds of drivers? Looks like it is at least used for "module" and "ebpf".

Reply:
The loader init container is disabled when running running with `modern-bpf`, you can see the [templating in the Helm chart here](https://github.com/falcosecurity/charts/blob/falco-3.3.0/falco/templates/pod-template.tpl#L229-L233). It is also mentioned [here](https://github.com/falcosecurity/charts/tree/falco-3.3.0/falco#about-drivers) that `modern-bpf` does not require the loader. So the Helmfile templating here should just be ignored when running with `kind: modern-bpf`